schedule/functional/redis.yaml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
This is not necessary. Just use `INCLUDE_MODULES=bootloader_start,boot_to_desktop,redis` when you're running your verifications ;)

Reply:
yes, thanks!